Finalizes install.sh tails script

conorsch opened the pull request:
Implements eleventh hour cleanup for the new `tails_files/install.sh` script (originally provided in #1309). Briefly:

* removes verbose logging messages to avoid cluttering terminal output
* silences STDERR for `zenity` prompts to avoid cluttering terminal outut
* preserves informative messaging around needing to prompt, but redirects to STDERR rather than STDOUT.

See #1316 for verbose explanations of the rationale behind these changes. Closes #1316.

garrettr commented on tails_files/install.sh:
```diff
@@ -209,23 +209,20 @@ function lookup_document_aths_url()
   # Couldn't find it anywhere. We'll have to prompt!
   else
-    echo "Could not find Document Interface ATHS info, prompting interactively..."
+    echo "Could not find Document Interface ATHS info, prompting interactively..." 1>&2
```
What does this change do, and why?

ageis replied:
Redirects output to STDERR...

conorsch replied:
By redirecting the verbose logging message to STDERR, we ensure that the echoed text does not interfere with any of the variable lookups. All of the lookup functions use STDOUT capture (via `$()`) to return config information. This is mostly a defensive measure to ensure that outputs are not mixed.

garrettr replied:
@conorsch Thanks for the explanation, very helpful.

conorsch commented:
Fixed the zenity commit and rebased on top of `release/0.3.7`. My commits in the issue and PR about the necessity of redirecting only STDERR and not STDOUT were on point. During review, @garrettr noticed that not all calls to zenity were following that pragma.
